nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -383,7 +383,3 @@ nifi.performance.tracking.percentage=${nifi.performance.tracking.percentage}
 #   nifi.nar.library.provider.hdfs.kerberos.principal=nifi@NIFI.COM
 #   nifi.nar.library.provider.hdfs.kerberos.keytab=/path/to/nifi.keytab
 #   nifi.nar.library.provider.hdfs.kerberos.password=
-#
-# Example NiFi Registry NAR Provider:
-#   nifi.nar.library.provider.nifi-registry.implementation=org.apache.nifi.registry.extension.NiFiRegistryNarProvider
-#   nifi.nar.library.provider.nifi-registry.url=http://localhost:18080
```

Comment:
Could we leave this block in the properties file, but just fix the class name from `NiFiRegistryNarProvider` to `NiFiRegistryExternalResourceProvider` ?

Reply:
Good point, will make the adjustment.

Reply:
@bbende Pushed an update restoring the Registry example using the preferred Provider class name.